### DISCUSSION TITLE
Feature request: choose another language for date formats

### DISCUSSION THREAD

**Discussion by MelodyDuplaix** [coddingtonbear/obsidian-local-rest-api#72] [Ideas]
My daily notes are in 08-20-2023 di format. I wanted to set up a template to record a url in my daily note. Is there a solution? If not, would it be possible to add the ability to change the language?

**Comment by coddingtonbear** [comment]
Good news, @MelodyDuplaix -- although this plugin can interact with your daily notes in Obsidian, this extension has nothing to do with anything like date formats in use.  It sounds like you just need to configure Obsidian's "Daily Notes" plugin to use the date format you want.

**Reply by MelodyDuplaix** [reply]
But, I already change the format in the Obsidian daily notes in my format, and the obsidian web format remains false, it does not take this parameter into account. Or is it a bug.

**Reply by coddingtonbear** [reply]
I'm afraid it's not likely to be a bug in this library, but it could be a bug in the interface we use for generating these notes (https://github.com/liamcain/obsidian-daily-notes-interface/), and it does look like there's a suspiciously-named bug posted there that could be related: https://github.com/liamcain/obsidian-daily-notes-interface/issues/24 .  It doesn't look like that repository is actively maintained, though, so I'm not sure how much luck you'd have by starting a discussion there.  Maybe it's worth a shot, though?
